python/private/whl_target_platform.bzl

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A starlark implementation of the wheel platform tag parsing to get the target platform.
+"""
+
+_cpu_aliases = {
+    "x86_64": "x86_64",
+    "x86_32": "x86_32",
+    "aarch64": "aarch64",
+    "ppc": "ppc",
+    "s390x": "s390x",
+    # aliases
+    "amd64": "x86_64",
+    "arm64": "aarch64",
+    "i386": "x86_32",
+    "i686": "x86_32",
+    "x86": "x86_32",
+    "ppc64le": "ppc",
+}  # buildifier: disable=unsorted-dict-items
+
+_os_prefixes = {
+    "linux": "linux",
+    "manylinux": "linux",
+    "musllinux": "linux",
+    "macos": "osx",
+    "win": "windows",
+}  # buildifier: disable=unsorted-dict-items
+
+def whl_target_platform(tag):
```

Comment:
The name/signature/return value of this function is a bit confusing.

It has a singular name, but is documented to return a list of vales, but only returns a single value. Can this be reconciled, please?

Reply:
Renamed to plural.